views/dataset-data-connector-page.twig

```diff
@@ -67,7 +67,7 @@
                                 </div>
                             </div>
 
-                            {{ forms.button("Save"|trans, "submit", null, null, null, "btn-success") }}
+                            {{ forms.button("Save"|trans, "submit", null, null, null, "btn-success " ~ (dataSet.dataConnectorSource != 'user_defined' ? 'disabled' : ''), disabled) }}
```

Comment:
Should we hide it as well?

Reply:
Oh, it's already within the div that gets hidden if the source is not user-defined, so there's no need to explicitly hide the button within that div.

But I forgot to remove the disabled parameter (it was a test to see if it works), so I'll remove that one.
` ''), disabled) }}`